data/elan-2ad9.tablet

```diff
@@ -0,0 +1,14 @@
+# ELAN touchscreen/pen sensor in the HP Probook x360 435 G7 
```

Comment:
can you submit your sysinfo to https://github.com/linuxwacom/wacom-hid-descriptors/ and link to it here please. Will help with future debugging of the device.